lib/src/testing/utils/detected_runtime_js_interop.dart

```diff
@@ -12,8 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// ignore: avoid_web_libraries_in_flutter
-import 'dart:html' show window;
+import 'dart:js_interop';
+
+@JS('window.navigator.userAgent')
+external String get userAgent;
```

Comment:
Can we do this in a separate PR, I'm guessing that this change won't break existing tests and it would be easy to land in a separate PR -- thus, making this PR smaller.

Reply:
I made a separate PR.
https://github.com/google/webcrypto.dart/pull/180